# A-Bioinformatics-Paper
As a new postgraduate, this repository contains a collection of papers on bioinformatics and artificial intelligence to facilitate my learning and document my progress.
### Interaction Prediction
1. [AAAI 2024] **PSC-CPI: Multi-Scale Protein Sequence-Structure Contrasting for Efficient and Generalizable Compound-Protein Interaction Prediction**[[paper]](https://arxiv.org/pdf/2402.08198)
2. [Nature Communications 2024] **PROFSA: SELF-SUPERVISED POCKET PRETRAINING VIA PROTEIN FRAGMENT-SURROUNDINGS ALIGNMEN**[[paper]](https://arxiv.org/pdf/2310.07229)
3. [NeurIPS 2024] **FlexMol: A Flexible Toolkit for Benchmarking Molecular Relational Learning**[[paper]](https://steven51516.github.io/assets/pdf/flexmol.pdf)
4. [ICML 2024] **Learning to Predict Mutational Effects of Protein-Protein Interactions by Microenvironment-aware Hierarchical Prompt Learning**[[paper]](https://arxiv.org/pdf/2405.10348)
5. [ICLR 2024] **MAPE-PPI: TOWARDS EFFECTIVE AND EFFICIENT PROTEIN-PROTEIN INTERACTION PREDICTION VIA MICROENVIRONMENT-AWARE PROTEIN EMBEDDING**[[paper]](https://openreview.net/pdf?id=itGkF993gz)

### Protein Representation Learning
1. [CVPR 2024] **Clustering for Protein Representation Learning**[[paper]](https://arxiv.org/pdf/2404.00254)
2. [ECCV 2024] **Unsupervised Representation Learning by Balanced Self Attention Matching**[[paper]](https://www.ecva.net/papers/eccv_2024/papers_ECCV/papers/11230.pdf)
3. [ICLR 2024] **Learning Complete Protein Representation by Dynamically Coupling of Sequence and Structure**[[paper]](https://openreview.net/pdf?id=0e5uOaJxo1)
4. [ICML 2024] **ESM All-Atom: Multi-Scale Protein Language Model for Unified Molecular Modeling**[[paper]](https://arxiv.org/abs/2403.12995)[[tweet]](https://air.tsinghua.edu.cn/info/1007/2271.htm)
5. [NeurIPS 2024] **Learning Complete Protein Representation by Dynamically Coupling of Sequence and Structure**[[paper]](https://neurips.cc/virtual/2024/poster/96915)
6. [NeurIPS 2024] **MMSite: A Multi-modal Framework for the Identification of Active Sites in Proteins**[[paper]](https://openreview.net/pdf?id=XHdwlbNSVb)
7. [NeurIPS 2024] **ProtGO: Function-Guided Protein Modeling for Unified Representation Learning**[[paper]](https://openreview.net/pdf?id=0oUutV92YF)
8. [ICLR 2021] **Learning from protein structure with geometric vector perceptrons**[[paper]](https://arxiv.org/pdf/2009.01411)
9. [NeurIPS 2021] **Multi-Scale Representation Learning on Proteins**[[paper]](https://openreview.net/pdf?id=-xEk43f_EO6)
### Molecular Property Prediction(MMP)
1. [AAAI 2024] **Improving PTM Site Prediction by Coupling of Multi-Granularity Structure and Multi-Scale Sequence Representation**[[paper]](https://arxiv.org/pdf/2401.10211)
2. [ICLR 2024] **PROTEIN-LIGAND BINDING REPRESENTATION LEARNING FROM FINE-GRAINED INTERACTIONS**[[paper]](https://openreview.net/pdf?id=AXbN2qMNiW)
3. [ICLR 2024] **LEARNING TO DESIGN PROTEIN–PROTEIN INTERACTIONS WITH ENHANCED GENERALIZATION**[[paper]](https://openreview.net/pdf?id=xcMmebCT7s)
4. [ICLR 2024] **EVALUATING REPRESENTATION LEARNING ON THE PROTEIN STRUCTURE UNIVERSE**[[paper]](https://openreview.net/pdf?id=sTYuRVrdK3)
5. [ICLR 2024] **PRE-TRAINING SEQUENCE, STRUCTURE, AND SURFACE FEATURES FOR COMPREHENSIVE PROTEIN REPRESENTATION LEARNING**[[paper]](https://openreview.net/pdf?id=BEH4mGo7zP)
6. [ICLR 2024] **SaProt: PROTEIN LANGUAGE MODELING WITH STRUCTURE-AWARE VOCABULARY**[[paper]](https://openreview.net/pdf?id=6MRm3G4NiU)[[tweet]](https://blog.csdn.net/qq_61219755/article/details/144692894)
7. [ICML 2024] **Learning to Predict Mutational Effects of Protein-Protein Interactions by Microenvironment-aware Hierarchical Prompt Learning**[[paper]](https://openreview.net/pdf?id=g89jAdrnAF)
8. [ICML 2024] **Pre-Training Protein Bi-level Representation Through Span Mask Strategy On 3D Protein Chains**[[paper]](https://openreview.net/pdf?id=qY63FnLuJ1)
9. [NeurIPS 2024] **S-MolSearch: 3D Semi-supervised Contrastive Learning for Bioactive Molecule Search**[[paper]](https://arxiv.org/pdf/2409.07462)
10. [NeurIPS 2018] **CheMixNet: Mixed DNN Architectures for Predicting Chemical Properties using Multiple Molecular Representations**[[paper]](https://arxiv.org/pdf/1811.08283)
### AI Drug Discover(AIDD)
1. [NeurIPS 2023] **Understanding the Limitations of Deep Models for Molecular Property Prediction: Insights and Solutions**[[paper]](https://openreview.net/pdf?id=NLFqlDeuzt)
2. **ACross-Field Fusion Strategy for Drug–Target Interaction Prediction**[[paper]](https://arxiv.org/pdf/2405.14545)
### Protein Sequencing
1. [NeurIPS 2024] **Towards Robust De Novo Peptide Sequencing in Proteomics against Data Biases**[[paper]](https://openreview.net/pdf?id=0zfUiSX5si)
### Graph Contrastive Learning
1. [Advanced Science 2024] **Cross-Modal Graph Contrastive Learning with Cellular Images**[[paper]](https://onlinelibrary.wiley.com/doi/pdf/10.1002/advs.202404845)
### Predicting Protein Function
1. [Nature communitications 2025] **DPFunc: accurately predicting protein function via deep learning with domain-guided structure information**[[paper]](https://pubmed.ncbi.nlm.nih.gov/39746897/)
### Overview
1. **Advances of Deep Learning in Protein Science: A Comprehensive Survey**[[paper]](https://arxiv.org/pdf/2403.05314)
### Retrieval
1. [Nature Communications 2024] **DrugCLIP: Contrastive Protein-Molecule Representation Learning for Virtual Screening**[[paper]](https://proceedings.neurips.cc/paper_files/paper/2023/file/8bd31288ad8e9a31d519fdeede7ee47d-Paper-Conference.pdf)

### Benchmark
1. [NeurIPS 2024] **CARE:a Benchmark Suite for the Classification and Retrieval of Enzymes**[[paper]](https://openreview.net/pdf?id=PFwlw9bnAr)
2. [NeurIPS 2024] **ReactZyme: A Benchmark for Enzyme-Reaction Prediction**[[paper]](https://zhenglab.sjtu.edu.cn/uploadfile/ueditor/file/202410/1728892818c60954.pdf)
3. [NeurIPS 2024] **AsEP: Benchmarking Deep Learning Methods for Antibody-specific Epitope Prediction**[[paper]](https://arxiv.org/pdf/2407.18184)
4. [Bioinformatics 2021] **PPB-Affinity**[[code]](https://github.com/ChenPy00/PPB-Affinity?tab=readme-ov-file)
5. [Nature Methods 2021] **Building datasets and benchmarks for deep learning on protein structures**[[paper]](https://proceedings.neurips.cc/paper_files/paper/2023/file/b6167294ed3d6fc61e11e1592ce5cb77-Paper-Datasets_and_Benchmarks.pdf)
### NEWS
1. [ICLR 2024] **Decoupling Weighing and Selecting for Integrating Multiple Graph Pre-training Tasks**[[paper]](https://arxiv.org/pdf/2403.01400)
2. [NeurIPS 2024] **FlexSBDD: Structure-Based Drug Design with Flexible Protein Modeling**[[paper]](https://arxiv.org/pdf/2409.19645)
3. [NeurIPS 2024] **Enhancing Protein Mutation Effect Prediction through a Retrieval-Augmented Framework**[[paper]](https://openreview.net/pdf?id=LgeHswiWef)
4. [NeurIPS 2024] **Kermut: Composite kernel regression for protein variant effects**[[paper]](https://arxiv.org/pdf/2407.00002)
5. [NeurIPS 2024] **MutaPLM: Protein Language Modeling for Mutation Explanation and Engineering**[[paper]](https://arxiv.org/pdf/2410.22949)
6. [NeurIPS 2024] **Evolution-Inspired Loss Functions for Protein Representation Learning**[[paper]](https://openreview.net/pdf?id=y5L8W0KRUX)
7. [NeurIPS 2024] **MSAGPT:Neural Prompting Protein Structure Prediction via MSA Generative Pre-Training**[[paper]](https://arxiv.org/pdf/2406.05347)
8. [NeurIPS 2024] **Ultrafast classical phylogenetic method beats large protein language models on variant effect prediction**[[paper]](https://openreview.net/pdf?id=H7mENkYB2J)
9. #[ICLR 2025] **CROSS THE GAP: EXPOSING THE INTRA-MODAL MISALIGNMENT IN CLIP VIA MODALITY INVERSION**[[paper]](https://arxiv.org/pdf/2502.04263?)
10. #[ICCV 2023] **Zero-Shot Composed Image Retrieval with Textual Inversion**[[paper]](https://openaccess.thecvf.com/content/ICCV2023/papers/Baldrati_Zero-Shot_Composed_Image_Retrieval_with_Textual_Inversion_ICCV_2023_paper.pdf)
11. [BIB 2025] **ProteinF3S: boosting enzyme function prediction by fusing protein sequence, structure, and surface**[[paper]](https://academic.oup.com/bib/article/26/1/bbae695/7942219?login=false)
